src/main/java/org/folio/service/profiles/mappingprofile/MappingProfileService.java

```diff
@@ -56,16 +56,24 @@
   /**
    * Gets default {@link MappingProfile}
    *
-   * @param tenantId         tenant id
+   * @param tenantId tenant id
    * @return future with default {@link MappingProfile}
    */
   Future<MappingProfile> getDefault(String tenantId);
 
+//  /**
+//   * Check is mapping profile default or not {@link MappingProfile}
+//   *
```

Comment:
Looks like smth wrong here with the docs)

Reply:
done